.github/workflows/ci.yml

```diff
@@ -0,0 +1,30 @@
+name: ci
+
+on:
+  push:
+    branches: master
+
+jobs:
+  main:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Setup Docker Buildx
+        uses: docker/setup-buildx-action@v1
+      - name: Login to DockerHub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+      - name: Build and push
+        id: docker_build
+        uses: docker/build-push-action@v2
+        with:
+          context: .
+          file: ./Dockerfile
+          platforms: linux/amd64,linux/arm/v6,linux/arm/v7,linux/arm64
+          push: true
+          tags: biosmarcel/scribble.rs:latest
```

Comment:
It's actually `scribble-rs/scribble.rs`, should've mentioned that right away, sorry.

Reply:
I belive tag is based on https://hub.docker.com/r/biosmarcel/scribble.rs

Reply:
Oh I see. Sorry, wasn't thinking, was just reading my name and thought: errr, wrong ^^

Reply:
Something like this could work: https://github.community/t/is-there-a-way-to-depend-on-actions-from-another-file/16620

Reply:
Maybe we can also just run the tests once on linux before building. I just thought we'd waste as little github resources as possible this way :D